uint32 button_index
bool bool_data
int32 int_data
float64 float_data
string string_data
bool shouldCheckForAgentID
uint32[] agentIDs